src/thrift_parser.yrl

```diff
@@ -0,0 +1,238 @@
+
+Nonterminals
```

Comment:
I sort of like how the Elixir grammar groups these into lines of multiple names:

```
Nonterminals
  program
  headers header
  definitions definition
...
```